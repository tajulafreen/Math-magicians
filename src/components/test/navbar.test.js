import React from 'react';
import renderWithRouter from '../test-utils';
import Navigation from '../navbar';
import '@testing-library/jest-dom/extend-expect';

describe('Navigation component', () => {
  it('should match the snapshot', () => {
    const { container } = renderWithRouter(<Navigation />);
    expect(container).toMatchSnapshot();
  });
});
