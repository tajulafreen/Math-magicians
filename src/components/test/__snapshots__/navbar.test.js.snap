// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation component should match the snapshot 1`] = `
<div>
  <header>
    <h1>
      Math Magician
    </h1>
    <nav>
      <a
        href="/"
      >
        Home
      </a>
      <a
        href="/calculator"
      >
        Calculator
      </a>
      <a
        href="/quotes"
      >
        Quotes
      </a>
    </nav>
  </header>
</div>
`;
